app/src/test/java/io/apicurio/registry/noprofile/Ticket1062Test.java

```diff
@@ -0,0 +1,157 @@
+            "            \"state\",\n" +
+            "            \"postalCode\"\n" +
+            "          ]\n" +
+            "        }";
+
+    private static final String CONTACT_SCHEMA = "        {\n" +
+            "          \"$schema\": \"https://json-schema.org/draft/2019-09/schema\",\n" +
+            "          \"$id\": \"contact.schema.json\",\n" +
+            "          \"title\": \"Contact\",\n" +
+            "          \"description\": \"Schema for a contact\",\n" +
+            "          \"type\": \"object\",\n" +
+            "          \"properties\": {\n" +
+            "            \"person\": {\n" +
+            "              \"$ref\": \"./person.schema.json\"\n" +
+            "            },\n" +
+            "            \"address\": {\n" +
+            "              \"$ref\": \"./address.schema.json\"\n" +
+            "            },\n" +
+            "            \"email\": {\n" +
+            "              \"type\": \"string\",\n" +
+            "              \"format\": \"email\",\n" +
+            "              \"description\": \"The contact's email address.\"\n" +
+            "            }\n" +
+            "          },\n" +
+            "          \"required\": [\n" +
+            "            \"person\",\n" +
+            "            \"address\",\n" +
+            "            \"email\"\n" +
+            "          ]\n" +
+            "        }";
+
+    @Test
+    public void testDeleteArtifactWithInboundRefs() throws Exception {
+        String groupId = "default";
+        String addressId = TestUtils.generateArtifactId();
+        String personId = TestUtils.generateArtifactId();
+        String contactId = TestUtils.generateArtifactId();
+
+        // Create the address schema
+        InputStream addressData = toStream(ADDRESS_SCHEMA);
+        clientV2.createArtifact(groupId, addressId, addressData);
+
+        // Create the person schema
+        InputStream personData = toStream(PERSON_SCHEMA);
+        clientV2.createArtifact(groupId, personId, personData);
+
+        // Create the contact schema (with references)
+        InputStream contactData = toStream(CONTACT_SCHEMA);
+        clientV2.createArtifact(groupId, contactId, contactData, List.of(
+                ref("./person.schema.json", groupId, personId, "1"),
+                ref("./address.schema.json", groupId, addressId, "1")
+        ));
+
+        // Try to delete the "address" artifact, version 1 - using the ccompat API - should fail
+        Assertions.assertThrows(RestClientException.class, () -> {
+            confluentClient.deleteSchemaVersion(Map.of(), addressId, "1");
+        });
+        VersionMetaData addressVMD = clientV2.getArtifactVersionMetaData(groupId, addressId, "1");
+        Assertions.assertNotNull(addressVMD);
+        Assertions.assertEquals(addressId, addressVMD.getId());
+
+        // Try to delete the "contact" artifact itself - using the ccompat API - should fail
```

Comment:
Typo: Should be "address". That confused me for a moment:)

Reply:
Nice catch. :) Fixed.